entity-management-web/src/test/java/eu/europeana/entitymanagement/util/EntityDataReconceliationTest.java

```diff
@@ -0,0 +1,93 @@
+package eu.europeana.entitymanagement.util;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.Test;
+
+import eu.europeana.entitymanagement.definitions.model.impl.BaseTimespan;
+import eu.europeana.entitymanagement.definitions.model.impl.BaseWebResource;
+import eu.europeana.entitymanagement.web.service.impl.MetisDereferenceService;
+
+/**
+ * JUnit tests to merge the data from a different entity (a so called entity proxy) into the given entity
+ */
+public class EntityDataReconceliationTest {
```

Comment:
This test methods must be part of the Service tests, and the Test classes must follow the naming conventions: <ServiceNaname>Test

Reply:
Done.